pkg/processor/build/builder_test.go

```diff
@@ -672,6 +702,18 @@ func (suite *testSuite) testResolveFunctionPathArchive(buildConfiguration functi
 	suite.Equal(`def handler(context, event):
 	return "hello world"
 `, string(decompressedPythonFileContent))
+
+	suite.tearDownAfterResolveFunctionPath()
```

Comment:
these custom `tearDown` and `setUp` tells me they needs their own test suite

Reply:
not needed here, renamed `setUp` to a more accurate name - `mockArchiveFileURLEndpoint`